src/store.test.js

```diff
@@ -0,0 +1,55 @@
+import {mount} from 'enzyme';
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import Overridable, {OverridableContext, overrideStore} from '.'; // export from index.js to test it
+
+class ExampleComponent extends Component {
+  static propTypes = {
+    title: PropTypes.string.isRequired,
+  };
+
+  render() {
+    const {title} = this.props;
+    return <h1>{title}</h1>;
+  }
+}
+
+const CMP_ID = 'ExampleComponent';
+const OverridableExampleComponent = Overridable.component(CMP_ID, ExampleComponent);
+
+describe('Tests for store utility object.', () => {
+  beforeEach(() => {
+    overrideStore.clear();
+  });
+
+  test('it should render the cmp added to the global store instead of default cmp with id `ExampleComponent`', () => {
+    const NewComponent = () => <h2>Stored in the global store</h2>;
+    overrideStore.add(CMP_ID, NewComponent);
+
+    const mounted = mount(
+      <OverridableContext.Provider value={overrideStore.getAll()}>
+        <OverridableExampleComponent title="Ignored title" color="red">
+          <ul>
+            <li>It should NOT appear!</li>
+          </ul>
```

Comment:
Are you missing `children` in `ExampleComponent`? Otherwise I think this will never be taken into account. It could also be completely removed.

Reply:
Thanks, fixed!